strings/aho_corasick.py

```diff
@@ -1,10 +1,17 @@
 from collections import deque
-from typing import Dict, List, Union
+from typing import Deque, TypedDict, Union
+
+
+class type_of_dict(TypedDict):
```

Comment:
Could you change the name of the class to something meaningful and also in CamelCase ?

Reply:
Yes sure, I have renamed it to DictionaryOfState, please do tell if it needs to be renamed again😅.